closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -887,101 +997,102 @@ boolean isRestrictedProviderAllowed(String providerName) {
         /**
          * List audit info if userSecurityAudit is true, default as false.
```

Comment:
userSecurityAudit is no longer used, pls update the comment.

Reply:
Same.